wear/src/main/java/io/homeassistant/companion/android/tiles/ShortcutsTile.kt

```diff
@@ -211,10 +219,34 @@ class ShortcutsTile : TileService() {
             // Add icon
             LayoutElementBuilders.Image.Builder()
                 .setResourceId(entity.entityId)
-                .setWidth(dp(ICON_SIZE))
-                .setHeight(dp(ICON_SIZE))
+                .setWidth(dp(iconSize))
+                .setHeight(dp(iconSize))
                 .build()
         )
+        if (showLabels) {
+            addContent(
+                LayoutElementBuilders.Arc.Builder()
+                    .addContent(
+                        LayoutElementBuilders.ArcText.Builder()
+                            .setText(entity.friendlyName)
```

Comment:
I know this will probably vary from device to device to device but should we consider some type of character limit here? What happens if a user has a large friendly name that wraps around the icon?

Reply:
Good question, I'll run a test

Reply:
![Screenshot_20220115_104340_sysui](https://user-images.githubusercontent.com/34075651/149617512-374a3228-8107-4e11-b5c6-aeb0d1672b8c.png)

The one on the bottom left is now clipping. So it's not fully overlapping but a tiny bit weird

Reply:
fair amount of characters are shown probably ok to let the user decide if they want to change their name or deal with it